sql_driver/lib.go

```diff
@@ -16,7 +19,17 @@ func (d *LibsqlDriver) Open(dbPath string) (driver.Conn, error) {
 	if strings.HasPrefix(dbPath, "file:") {
```

Comment:
The argument to this function is a URL, not a path, so the name of the argument should reflect that.
(This comment should be on line 18, but Github doesn't allow that.)

Reply:
done